cloud/amazon/ec2_elb_lb.py

```diff
@@ -344,6 +351,13 @@
 except ImportError:
     HAS_BOTO = False
 
+try:
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
```

Comment:
the module will not break when people are not declaring "tags" and do not have boto3.  if "tags" option is declared, and boto3 is not available, mod fails as fast as possible.